project-properties.gradle

```diff
@@ -10,7 +10,7 @@ project.ext {
     releaseMode = project.hasProperty("releaseMode")
     hasDBSchema = project.hasProperty("DB_SCHEMA_POSTGRES")
     scriptsUrl = commonScriptsUrl + (releaseMode ? getProperty('scripts.version') : 'develop')
-    migrationsUrl = migrationsScriptsUrl + (releaseMode ? getProperty('migrations.version') : 'develop')
+    migrationsUrl = migrationsScriptsUrl + (releaseMode ? getProperty('migrations.version') : 'EPMRPP-90160')
```

Comment:
Need to change to develop as EPMRPP-90160 branch was merged

Reply:
feature/orgs actually. Already done as part of another PR